pkgimage.mk

```diff
@@ -41,6 +41,19 @@ $$(BUILDDIR)/stdlib/$1.debug.image: $$($1_SRCS) $$(addsuffix .debug.image,$$(add
 	@$$(call PRINT_JULIA, $$(call spawn,$$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'Base.compilecache(Base.identify_package("$1"))')
 	@$$(call PRINT_JULIA, $$(call spawn,$$(JULIA_EXECUTABLE)) --startup-file=no -e 'Base.compilecache(Base.identify_package("$1"))')
 	touch $$@
+endif
+ifneq ($(filter $(1),$(PKG_EXTS)),)
+# This is weird. It set up for multiple Pkg exts because that suits the structure here better
+# but it hard codes the deps and `import REPL, Pkg`
+$(BUILDDIR)/stdlib/REPLExt.release.image: $$(REPLExt_SRCS) $(BUILDDIR)/stdlib/Pkg.release.image $(BUILDDIR)/stdlib/REPL.release.image
+	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg')
+	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg')
+	touch $$@
+$(BUILDDIR)/stdlib/REPLExt.debug.image: $$(REPLExt_SRCS) $(BUILDDIR)/stdlib/Pkg.debug.image $(BUILDDIR)/stdlib/REPL.debug.image
+	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg')
+	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg')
+	touch $$@
+endif
```

Comment:
```suggestion
ifneq ($(filter $(1),$(PKG_EXTS)),)
# This is weird. It set up for multiple Pkg exts because that suits the structure here better
# but it hard codes the deps and `import REPL, Pkg`
$$(BUILDDIR)/stdlib/REPLExt.release.image: $$(REPLExt_SRCS) $$(BUILDDIR)/stdlib/Pkg.release.image $$(BUILDDIR)/stdlib/REPL.release.image
	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg')
	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg')
	touch $$@
$$(BUILDDIR)/stdlib/REPLExt.debug.image: $$(REPLExt_SRCS) $(BUILDDIR)/stdlib/Pkg.debug.image $$(BUILDDIR)/stdlib/REPL.debug.image
	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg')
	@$$(call PRINT_JULIA, $$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg')
	touch $$@
endif
```